hal/include/HAL/HAL.h

```diff
@@ -49,6 +54,7 @@ int32_t HAL_GetFPGAVersion(int32_t* status);
 int64_t HAL_GetFPGARevision(int32_t* status);
 uint64_t HAL_GetFPGATime(int32_t* status);
 
+HAL_RuntimeType HAL_GetRuntimeType();
```

Comment:
There are other places in the HAL we take and/or return an enum directly as well. Should we change those as well?